rreplay.py

```diff
@@ -49,6 +54,7 @@ def main():
         exit(1)
 
     # instantiate new SafeConfigParser, read path to config
+    print("-- Begin parsing INI configuration file")
```

Comment:
We want to minimize always-on printing in order to keep things quiet once we integrate this into a CI/CD type situation